lib/markdown_videos/renderer.rb

```diff
@@ -13,42 +13,26 @@ def initialize(string, options = {})
     end
 
     def render
-      MarkdownVideos::SERVICES.keys.each do |service|
-        @string = render_service(service)
-      end
-      @string
-    end
+      @string.gsub(/!\[([^\]]*)\]\(([^)]+)\)/) do
```

Comment:
Oh right, I was thinking `gsub!`. I will use `markdown_text`, it rightly makes more sense, thanks